### PR TITLE
release-notes: update for 20231004 releases

### PR DESCRIPTION
**Pull request by dustymabe**

### CODE REVIEW

File: release-notes/stable.yml [hunk 1]
```diff
@@ -1,4 +1,7 @@
 releases:
+  38.20230918.3.0:
+      - text: "New Package Request: gvisor-tap-vsock-gvforwarder"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1557"
```
Comment (marmijo) [inline release-notes/stable.yml:4]:
```suggestion
  38.20230918.3.0:
    issues:
      - text: "New Package Request: gvisor-tap-vsock-gvforwarder"
        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1557"
```

File: release-notes/testing.yml [hunk 2]
```diff
@@ -1,4 +1,7 @@
 releases:
+  38.20231002.2.2:
+      - text: "CVE-2023-4911: Buffer overflow in glibc leading to privilege escalation"
+        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1591"
```
Comment (marmijo) [inline release-notes/testing.yml:4]:
```suggestion
  38.20231002.2.2:
    issues:
      - text: "CVE-2023-4911: Buffer overflow in glibc leading to privilege escalation"
        url: "https://github.com/coreos/fedora-coreos-tracker/issues/1591"
```

Comment (marmijo) [reply release-notes/testing.yml:4]: `issues:` wasnt included in the last entry either, https://github.com/coreos/fedora-coreos-streams/pull/773/files, and I'm not seeing a release notes entry for `38.20230918.2.0` on the download page.

**Review by marmijo — COMMENTED** [review]
Added two comments

**Comment by dustymabe** [comment]
ok I think I fixed the issues - LMK

**Review by marmijo — APPROVED** [review]
LGTM thanks! sorry I missed it during the last release cycle.

**Review by jlebon — APPROVED** [review]

**Pull request merged**